public/js/pimcore/elementservice.js

```diff
@@ -83,9 +83,19 @@ pimcore.elementservice.deleteElementsComplete = function(options, response) {
 pimcore.elementservice.deleteElementCheckDependencyComplete = function (window, res, options) {
 
     try {
-        let message = res.batchDelete ? t('delete_message_batch') : t('delete_message');
+        let message = '';
+        if (res.batchDelete) {
+            message += sprintf(t('delete_message_batch'), res.itemResults.length) + "<br /><div style='display: block; padding-top: 10px'>";
+            res.itemResults.forEach(function (item) {
+                message += "<b style='display: block; text-align: center;'>\"" + htmlspecialchars(item.path + item.key) + "\"</b>";
```

Comment:
i'd suggest to change to `<li>`, and only make the `item.key` bold, because the path could be the same for each line, also avoiding alignment to center as well, because it's makes more easy to read "vertically", so not too much variation due the line length

```suggestion
                if (res.itemResults.length > 0) {
                    message += "<ul>";
                    res.itemResults.forEach(function (item) {
                        message += '<li>' + htmlspecialchars(item.path) + '<b>' + htmlspecialchars(item.key) + '</b></li>';
                    })
                    message += "</ul>";
                }
```

![image](https://github.com/pimcore/admin-ui-classic-bundle/assets/6014195/ea299ecd-25b8-46b6-bcd2-aec863a6eb40)